saleor/graphql/checkout/tests/mutations/test_checkout_add_promo_code.py

```diff
@@ -197,8 +197,11 @@ def test_checkout_add_voucher_code_by_token_with_external_shipment(
 def test_checkout_add_voucher_code_with_display_gross_prices(
     api_client, checkout_with_item, voucher, site_settings, monkeypatch
 ):
-    site_settings.display_gross_prices = True
-    site_settings.save()
+    channel = checkout_with_item.channel
+    tc = channel.tax_configuration
+    tc.display_gross_prices = True
+    tc.save()
```

Comment:
```suggestion
    tc.save(update_fields=["display_gross_prices"])
```

Reply:
in `test_discounts.py` line 32 and 54 as well